deploy.sh

```diff
@@ -148,15 +148,19 @@ build_app_padding () {
   ) | xxd -p -r > "${tab_folder}/padding.bin"
 }
 
-build_app () {
+comma_separated () {
   # Flatten the array
   # This is equivalent to the following python snippet: ' '.join(arr).replace(' ', ',')
-  local feature_list=$(IFS=$'\n'; echo "$@")
-  if [ "X${feature_list}" != "X" ]
+  local list=$(IFS=$'\n'; echo "$@")
+  if [ "X${list}" != "X" ]
   then
-    feature_list="${feature_list// /,}"
+    feature_list="${list// /,}"
   fi
+  echo ${list}
+}
 
+build_app () {
+  local feature_list="$(comma_separated $@)"
```

Comment:
This is usually a bad pattern (`"$@"` should be preferred to `$@`), but in that case we don't expect spaces in feature names. Also the `comma_separated` function does not handle it well neither. And this is hopefully temporary until this becomes a python script.